### PR TITLE
qa/crontab: Add weekly crimson-rados suites

### PR DESCRIPTION
**Pull request by NitzanMordhai**
This commit adds the Crimson-RADOS suite to the crontab jobs that run weekly, matching the priority of the RADOS

Signed-off-by: Nitzan Mordechai nmordech@redhat.com

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [x] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Review by Matan-B — COMMENTED** [review]
lgtm
Can you please un-filter Seastore and perf? We tend to filter when gating PRs that have no relevant changes to them.
Weekly runs should be useful to Seastore and perf as well to establish a baseline.

File: qa/crontab/teuthology-cronjobs [hunk 1]
```diff
@@ -78,9 +74,10 @@ TEUTHOLOGY_SUITE_ARGS="--non-interactive --newest=100 --ceph-repo=https://git.ce
 32 20 * * 4        $CW $SS      4 --ceph main --suite powercycle -p 950
 40 20 * * 5        $CW $SS      1 --ceph main --suite        rgw -p 950
 48 20 * * 6        $CW $SS      4 --ceph main --suite       krbd -p 950 --kernel testing
+56 20 * * 6        $CW $SS     64 --ceph main --suite crimson-rados -p 101 --flavor crimson
```
Comment (Matan-B) [inline qa/crontab/teuthology-cronjobs:77]: Should --force-priority be used with `-p 101`?

Comment (Matan-B) [reply qa/crontab/teuthology-cronjobs:77]: I'm not sure that we need a (64) subset for crimson-rados suite as it's quite limited already.

**Review by Matan-B — COMMENTED** [review]
Can we test this prior to merging?

File: qa/crontab/teuthology-cronjobs (hunk 1 above)
Comment (NitzanMordhai) [reply qa/crontab/teuthology-cronjobs:77]: > Should --force-priority be used with `-p 101`?

i matched it to rados suite, it looks like most of the time (with 900+ priority) the jobs were not scheduled at all.

Comment (NitzanMordhai) [reply qa/crontab/teuthology-cronjobs]: yes, --force-priority is needed. thanks

**Review by Matan-B — APPROVED** [review]

**Comment by Matan-B** [comment]
> @Matan-B all tests required are passed, I don't think any qa suites have a test against crontab..

@NitzanMordhai, I meant whether there is an option to test that the scheduling works as expected, since I'm not familiar enough with cronjobs.
We can either wait for an additional review or merge to verify this PR.

**Pull request merged**